src/features/smartSearch/components/filters/EmailClick/DisplayEmailClick.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box, Chip, Tooltip } from '@mui/material';
+
+import DisplayTimeFrame from '../DisplayTimeFrame';
+import { getTimeFrameWithConfig } from '../../utils';
+import messageIds from 'features/smartSearch/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import UnderlinedMsg from '../../UnderlinedMsg';
+import UnderlinedText from '../../UnderlinedText';
+import useCampaigns from 'features/campaigns/hooks/useCampaigns';
+import useEmails from 'features/emails/hooks/useEmails';
+import useLinks from 'features/emails/hooks/useLinks';
+import { useNumericRouteParams } from 'core/hooks';
+import { EMAIL_SELECT_SCOPE, LINK_SELECT_SCOPE } from '.';
+import {
+  EmailClickFilterConfig,
+  OPERATION,
+  SmartSearchFilterWithId,
+} from 'features/smartSearch/components/types';
+
+const localMessageIds = messageIds.filters.emailClick;
+
+interface DisplayEmailClickProps {
+  filter: SmartSearchFilterWithId<EmailClickFilterConfig>;
+}
+
+const DisplayEmailClick = ({ filter }: DisplayEmailClickProps): JSX.Element => {
+  const { orgId } = useNumericRouteParams();
+  const emails = useEmails(orgId).data || [];
+  const projects = useCampaigns(orgId).data || [];
+
+  const { config } = filter;
+  const { operator, campaign: projectId, email: emailId, links } = config;
+  const linksFuture = useLinks(orgId, filter.config?.email).data;
```

Comment:
I disagree with the name of this variable. It's called `linksFuture` (the important word here being "Future") which indicates to me that it's an `IFuture`, but it's not. It's the _data_ of a future, so it's really a link list.